pkg/standalone/standalone.go

```diff
@@ -1067,6 +1070,20 @@ func getPlacementImageWithTag(name, version string) string {
 	return fmt.Sprintf("%s:%s", name, version)
 }
 
+func checkFallbackImg(imageInfo daprImageInfo, fromDir string) bool {
+	if imageInfo.imageRegistryURL != "" || fromDir != "" {
+		return false
+	}
+	return imageInfo.imageRegistryName == githubContainerRegistryName
+}
+
+func useDefaultReg(privateReg, fromDir string) bool {
```

Comment:
Is `privateReg` supposed to be a URL? We can make the variable name reflect that.

Reply:
URL => URI